guides/V1_GetActionContext.md

```diff
@@ -0,0 +1,85 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'get-action-context'
+is: 'experimental'
+---
+
+
+# Get action context information 
+
+If you have registered a Client action handler callback for your App, then when your App is called through this callback, 
+it will receive an action handler context token. Your App needs to use this token with this API to extract the proper 
+action context.
+
+### Path
+```
+GET /v1/apps/{appId}/getActionContext/{actionHandlerContextToken}
+```
+**actionHandlerContextToken** has only 5 minutes lifetime, and can only be used once. It will be deleted once it is used. 
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Header**|Authorization required|Authorization header, in form of Bearer {access_token}. The access_token is issued as result of an oauth flow by Watson Work Services|String|
+|**Path**|appId required|Unique id of an app|String|
+|**Path**|actionHandlerContextToken required|token generated by caller before|String|
```

Comment:
see above picture.... sentence ends with `before` in strange way :-)

Perhaps a better description for this URL param is:

token previously generated by Watson Work Services for a client issuing this URL.